source/catalog.html

```diff
@@ -0,0 +1,385 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body class="page__body">
+  <header class="main-header">
+    <a class="main-header__logo">
+      <img class="main-header__logo-image" src="img/logo-desktop.svg" alt="Cat Energy" width="202" height="59">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+      <div class="main-nav__wrapper">
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item">
+            <a href="#">
+              Главная
+            </a>
+          </li>
+          <li class="site-list__item site-list__item--active">
+            <a>
+              Каталог продукции
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">
+              Подбор программы
+            </a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <main class="main-container">
+    <section class="catalog">
+      <div class="catalog__wrapper">
+        <h1 class="catalog__title">Каталог продукции</h1>
+        <ul class="catalog__product-list">
+          <li class="catalog__product-item">
+            <img src="img/catalog/cat-energy-1.png" alt="вкус курицы маленький" width="68" height="86">
+            <h2 class="catalog__title-product-mini">Cat Energy PRO 500 г</h2>
+            <table class="catalog__info">
+              <tr>
+                <td>
+                  Масса
+                </td>
+                <td>
+                  500 г
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  Вкус
+                </td>
+                <td>
+                  Курица
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  Цена
+                </td>
+                <td>
+                  700 ₽
+                </td>
+              </tr>
+            </table>
+            <a class="catalog__button" href="#">ЗАКАЗАТЬ</a>
```

Comment:
Лучше по обычным правилам писать, потом в css сделаем заглавными
```suggestion
            <a class="catalog__button" href="#">Заказать</a>
```